src/Generation/Generator/Renderer/Public/ParameterToNativeExpression/Converter/String.cs

```diff
@@ -10,6 +11,10 @@ public bool Supports(GirModel.AnyType type)
 
     public void Initialize(ParameterToNativeData parameter, IEnumerable<ParameterToNativeData> _)
     {
+        // This is tricky to auto-generate bindings for, since it can expect the
```

Comment:
Could you update this comment to something along the lines: `//Todo the caller needs to pass in some kind of Span<T> as a buffer which can be filled by the C function`